client/src/main/java/io/fabric8/docker/client/utils/ArchiveUtil.java

```diff
@@ -34,7 +35,7 @@ public static void tar(Path inputPath, Path outputPath) throws IOException {
 
         try (TarArchiveOutputStream tarArchiveOutputStream = buildTarStream(outputPath.toFile())) {
             if (!Files.isDirectory(inputPath)) {
-                TarArchiveEntry tarEntry = new TarArchiveEntry(inputPath.toFile());
+                TarArchiveEntry tarEntry = new TarArchiveEntry(inputPath.toFile().getName());
```

Comment:
If we create TarArchiveEntry using file in constructor, we have issue to set size while putting this TarArchiveEntry as it's setting all file information from file [header](https://sourcegraph.com/github.com/lookout/commons-compress@52dd5908e374973d69c51856b74d4d93d591c90a/-/blob/src/main/java/org/apache/commons/compress/archivers/tar/TarArchiveEntry.java#L307-308) which I observed in arquillian-cube tests.